portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,6 +44,7 @@
   private static final String content = "content";
   private static final String textInput = "text-input";
   private static final String quantity = "quantity";
```

Comment:
nit: I would suggest moving constants above the instance fields.

Also use caps for the constant names: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names